doc/_extensions/zephyr/vcs_link.py

```diff
@@ -60,22 +68,70 @@ def vcs_link_get_url(app: Sphinx, pagename: str) -> Optional[str]:
     return "/".join(
         [
             app.config.vcs_link_base_url,
+            mode,
+            app.config.vcs_link_version,
             found_prefix,
             app.env.project.doc2path(pagename, basedir=False),
         ]
     )
 
 
+def vcs_link_get_open_issue_url(app: Sphinx, pagename: str, sha1: str) -> Optional[str]:
+    """Link to open a new Github issue regarding "pagename" with title, body, and
+    labels already pre-filled with useful information.
+
+    Args:
+        app: Sphinx instance.
+        pagename: Page name (path).
+
+    Returns:
+        URL to open a new issue if applicable, None otherwise.
+    """
+
+    if not os.path.isfile(app.env.project.doc2path(pagename)):
+        return None
+
+    title = quote(f"[doc] Documentation issue in '{pagename}'")
+    labels = quote("area: Documentation")
+    body = quote(
+        dedent(
+            f"""\
+    **Describe the bug**
+
+    << Please describe the issue here >>
+    << You may also want to update the automatically generated issue title above. >>
+
+    **Environment**
+
+    * Page: `{pagename}`
+    * Version: {app.config.vcs_link_version}
+    * SHA-1: {sha1}
+    """
+        )
+    )
+
+    return f"{app.config.vcs_link_base_url}/issues/new?title={title}&labels={labels}&body={body}"
```

Comment:
nit: this extension has now become GH specific, so the "vcs" generic naming should be abandoned

Reply:
refactored both "vcs" and "git" extension in the latest update, please have another look, thx!